server/controllers/usage-controller.js

```diff
@@ -150,7 +150,7 @@ exports.getBookingStats = async (req, res) => {
       knex
         .select(
           { lecture_id: "sl.lecture_id" },
-          { lecture_name: "sl.lecture_name" },
+          //{ lecture_name: "sl.lecture_name" },
```

Comment:
Usually it's seen as a bad practice to commit files containing commented lines of code.